README.md

```diff
@@ -50,3 +82,15 @@
 
 ## Advance ML-on-Code awareness
 
+- Is it supposed to be MLonCode instead of ML-on-Code? - Vadim
+
+### ML team activities
+
+- [ ] Speak on two conferences
```

Comment:
ML Team doesn't have time in this Q3 for giving conferences.

Reply:
Have they agreed to any conferences already?
@vmarkovtsev

Reply:
I am personally speaking on PyData Mallorca in 2 days and ML4P on July 19th. There was no new conference submissions without Máximo's blessing.